x-pack/plugin/fleet/qa/rest/build.gradle

```diff
@@ -1,31 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-apply plugin: 'elasticsearch.internal-yaml-rest-test'
-apply plugin: 'elasticsearch.yaml-rest-compat-test'
-apply plugin: 'elasticsearch.internal-test-artifact'
-
-restResources {
-  restApi {
-    include '_common', 'bulk', 'cluster', 'nodes', 'indices', 'index', 'xpack', 'security', 'fleet'
-  }
-}
-
-artifacts {
-  restXpackTests(new File(projectDir, "src/yamlRestTest/resources/rest-api-spec/test"))
-}
-
-tasks.named('yamlRestTest') {
-  usesDefaultDistribution("to be triaged")
-}
-tasks.named('yamlRestCompatTest') {
-  usesDefaultDistribution("to be triaged")
-}
-if (buildParams.inFipsJvm){
```

Comment:
I think you forgot this to be ported to the fleet project build script?

Reply:
Looks like moving to the integ-test distro sorted this out.

Reply:
I've added the fips label to ensure we test this still works.